app/repository/repository-detail-controller.js

```diff
@@ -27,13 +34,13 @@ angular.module('repository-detail-controller', ['registry-services', 'app-mode-s
     $scope.getNextHref = function (){
       if($scope.maxTagsPage > $scope.tagsCurrentPage){
         var nextPageNumber = $scope.tagsCurrentPage + 1;
-        return '/repository/'+$scope.repository+'/'+ $scope.tagsPerPage +'/' +nextPageNumber;
+        return '/repository/'+$scope.repository+'?tagsPerPage='+ $scope.tagsPerPage +'&tagPage=' +nextPageNumber;
```

Comment:
OK, I:

- removed `:tagsPerPage` from the repository detail routes in 8e635d2421e0c089d7c20bc7c0c881412814c0ba.
- updated the unit tests to pass `tagsPerPage` as a query string parameter rather than in the route in 15d5cdb.